src/MakePotCommand.php

```diff
@@ -377,7 +398,17 @@ protected function makepot() {
 			}
 		}
 
-		return PotGenerator::toFile( $this->translations, $this->destination );
+		$result = PotGenerator::toFile( $this->translations, $this->destination );
+
+		$translations_count = count( $this->translations );
+
+		if ( 1 === $translations_count ) {
+			WP_CLI::debug( sprintf( 'Extracted %d string', count( $this->translations ) ), 'make-pot' );
+		} else {
+			WP_CLI::debug( sprintf( 'Extracted %d strings', count( $this->translations ) ), 'make-pot' );
```

Comment:
Reuse `$translations_count` here

Reply:
Maybe we should move over the `Inflector` class from `wp-cli/scaffold-package` for strings like this.
You could then do something along these lines:
```
WP_CLI::debug( sprintf(
   'Extracted %d %s',
   $translations_count,
   Utils\maybe_pluralize( 'string', $translations_count )
), 'make-pot' );
```

Reply:
See https://github.com/wp-cli/wp-cli/issues/4880

Reply:
Only now saw this. The same applies to the language command as well, so 👍